cmd/orb-server/startcmd/params.go

```diff
@@ -103,18 +104,23 @@ const (
 	casTypeEnvKey        = "CAS_TYPE"
 	casTypeFlagUsage     = "The type of the Content Addressable Storage (CAS). " +
 		"Supported options: local, ipfs. For local, the storage provider specified by " + databaseTypeFlagName +
-		" will be used. For ipfs, the node specified by " + ipfsURLFlagName + " will be used. " + commonEnvVarUsageText + casTypeEnvKey
+		" will be used. For ipfs, the node specified by " + ipfsURLFlagName +
+		" will be used. This is a required parameter. " + commonEnvVarUsageText + casTypeEnvKey
 
 	ipfsURLFlagName      = "ipfs-url"
 	ipfsURLFlagShorthand = "r"
 	ipfsURLEnvKey        = "IPFS_URL"
-	ipfsURLFlagUsage     = "The URL of the IPFS Content Addressable Storage (CAS). " + commonEnvVarUsageText + ipfsURLEnvKey
+	ipfsURLFlagUsage     = "Enables IPFS support. If set, this Orb server will use the node at the given URL. " +
+		"To use the public ipfs.io node, set this to https://ipfs.io (or http://ipfs.io). If using ipfs.io, " +
+		"then the CAS type flag must be set to local since the ipfs.io node is read-only. " +
+		"If the URL doesnt include a scheme, then HTTP will be used by default. " + commonEnvVarUsageText + ipfsURLEnvKey
```

Comment:
Typo - doesnt.

Reply:
Just saw this comment - I'll fix it in my next PR!